### PR TITLE
[ENH] Annotator - Add documentation

### PR DESCRIPTION
**Pull request by PrimozGodec**
##### Issue

Fixes https://github.com/biolab/orange3-text/issues/958

##### Description of changes
Add documentation for Annotated Corpus Map widget.

##### Includes
- [ ] Code changes
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: doc/widgets/annotator.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Documents maps can be enhanced with keyword annotations. This workflow embeds documents 
+in vector space, computes and annotates a t-SNE document map. The Annotator widget 
+identifies clusters on the map and annotates them with keywords representing a cluster.
+
+We load the **Proposal to government** dataset from 
+[documents repository](https://file.biolab.si/text-semantics/data/proposals-to-government-1k/) 
+with the **Import Documents** widget. In the **Corpus** widget, we set variables used 
+as text features and the title variable for display purposes.
+
+The **Docuemnt Embedding** widget embeds documents in the vector space for the
```
Comment (lenatr99) [inline doc/widgets/annotator.md:77]: Document*

File: doc/widgets/annotator.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+     If unchecked *Color* attribute is used for colouring.
+6. *Select, zoom, pan and zoom to fit* are the options for exploring the graph. 
+   The manual selection of data instances works as an angular/square selection tool. 
+   Double-click to move the projection. Scroll in or out for zoom.
+7. Plot with each point presenting documents. Documents are clustered according to 
+   the settings above, and clusters are marked with a hull. Over each cluster, 
+   the widget places the list of the most significant keywords.
+8. Changes are communicated automatically if *Send automatically* is ticked. 
+   Alternatively, press *Send*.
+9. Get hel, save the plot, add the plot to the report, set visual settings and get input
```
Comment (lenatr99) [inline doc/widgets/annotator.md:62]: help*

File: doc/widgets/annotator.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+We select t-SNE projection as the x and y variables. 
+And decide to use Gaussian mixture models for cluster identification. 
+The widget marks clusters with hulls and places keywords on each cluster. 
+
+We got clusters with different topics. In the left purple cluster, 
+we have documents connected to traffic, while the red cluster seems similar but 
+more focused on vehicles and registration. The yellow cluster includes documents 
+related to work and students.
+
+We can select a subset of documents of interest and observer them in the **Corpus Viewer**
```
Comment (lenatr99) [inline doc/widgets/annotator.md:90]: observe*

File: doc/widgets/annotator.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+[documents repository](https://file.biolab.si/text-semantics/data/proposals-to-government-1k/) 
+with the **Import Documents** widget. In the **Corpus** widget, we set variables used 
+as text features and the title variable for display purposes.
+
+The **Docuemnt Embedding** widget embeds documents in the vector space for the 
+**t-SNE** widget to project them in two-dimensional space. 
+
+**Annotated Corpus Map** plots documents on the two-dimensional space. 
+We select t-SNE projection as the x and y variables. 
+And decide to use Gaussian mixture models for cluster identification.
```
Comment (lenatr99) [inline doc/widgets/annotator.md:82]: We select t-SNE projection as the x and y variables and decide to use Gaussian mixture models for cluster identification.

**Review by lenatr99 — COMMENTED** [review]
Minor grammar mistakes.

**Comment by PrimozGodec** [comment]
@lenatr99 I fixed the grammatical errors and took a few texts from your pull request #958. The latest can be found as a separate commit authored by you.

**Pull request merged**